cob_relayboard/package.xml

```diff
@@ -10,22 +10,17 @@
   <author>Christian Connette</author>
   <maintainer email="mig@ipa.fhg.de">Matthias Gruhler</maintainer>
 
-
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>roscpp</build_depend>
-  <build_depend>rostest</build_depend>
   <build_depend>std_msgs</build_depend>
-  <build_depend>cob_utilities</build_depend>
   <build_depend>cob_msgs</build_depend>
 
   <run_depend>roscpp</run_depend>
-  <run_depend>rostest</run_depend>
+  <run_depend>rospy</run_depend>
   <run_depend>std_msgs</run_depend>
-  <run_depend>cob_utilities</run_depend>
   <run_depend>cob_msgs</run_depend>
 
-
   <export>
-    <cpp lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lSerialIO -lcob_relayboard" cflags="-I${prefix}/common/include"/>
+    <cpp cflags="-I${prefix}/common/include" lflags="-Wl,-rpath,${prefix}/common/lib -L${prefix}/common/lib -lSerialIO -lcob_relayboard"/>
```

Comment:
Do we need those flags at anymore? I guess this is a leftover from `rosbuild`...